lib/mpl_toolkits/tests/test_axes_grid1.py

```diff
@@ -84,6 +85,43 @@ def test_twin_axes_empty_and_removed():
     plt.subplots_adjust(wspace=0.5, hspace=1)
 
 
+@image_comparison(['twin_axes_both_with_units.png'])
```

Comment:
You are changing the main library, but only testing the axes_grid toolkit.  If this has ramifications for the normal `Axes.twinx/y` please test that as well, or even preferentially.

Reply:
We have just added a new optional parameter to it, which should not affect the existing functionality. The regression tests for the main library have all been passed as well.